### PR TITLE
Added writeValue and writeValueWithoutResponse.

### PR DESCRIPTION
**Pull request by cmumford**

### CODE REVIEW

File: characteristic_readwrite/web_app.js [hunk 1]
```diff
@@ -74,20 +74,43 @@ async function startTest() {
...
+
+    // writeValue is deprecated.
+    const fourthValue = 3000;
+    logInfo(`writing value ${fourthValue}...`);
+    result = await characteristic.writeValue(new Uint32Array([fourthValue]));
+    assertEquals(undefined, result);
+
+    dataview = await characteristic.readValue();
+    val = dataview.getUint32(0, /*littleEndian=*/true);
+    assertEquals(fourthValue, val, 'Incorrect second value');
```
Comment (reillyeon) [inline characteristic_readwrite/web_app.js]: "fourth"

**Review by reillyeon — APPROVED** [review]
This is useful as a sanity check for these functions but we should implement something which explicitly validates that the right Bluetooth messages are being sent, assuming that we can observe that from the Espruino side.

File: characteristic_readwrite/web_app.js (hunk 1 above)
Comment (cmumford) [reply characteristic_readwrite/web_app.js]: Sometimes I think I'm not qualified to use the clipboard.

**Comment by cmumford** [comment]
> This is useful as a sanity check for these functions but we should implement something which explicitly validates that the right Bluetooth messages are being sent, assuming that we can observe that from the Espruino side.

I experimented with raising exceptions in the device's onWrite callback, but that was never propagated back to the web app. I'll put that on my todo list.

**Pull request merged**